test/test_filt.py

```diff
@@ -209,7 +209,7 @@ def test_body(self):
 
     def test_method(self):
         q = self.req()
-        s = self.resp()
+        self.resp()
```

Comment:
`self.resp()` just creates a response object without side-effects. There is no point for keeping the whole line. (I suppose someone originally just copied `q=...; s=...` to each test)